tutorials/HDF5/Subset-HDF5-hyperspectral.Rmd

```diff
@@ -0,0 +1,322 @@
+preferably, RStudio loaded on your computer.
+
+### R Libraries to Install:
+
+* **rhdf5**: `install.packages("BiocManager")`, `BiocManager::install("rhdf5")`
+* **raster**: `install.packages('raster')`
+
+<a href="{{ site.baseurl }}/packages-in-r" target="_blank"> More on Packages in
+ R - Adapted from Software Carpentry.</a>
+
+### Data to Download
+The purpose of this tutorial is to reduce a large file (~`652Mb`) to a smaller size. The download linked here is the original large file, and therefore you may choose to skip this tutorial and download if you are on a slow internet connection or have file size limitations on your device.
+
+{% include/dataSubsets/_data_Imaging-Spec-Data-Full-Tile-H5-2020.html %}
+
+***
+{% include/_greyBox-wd-rscript.html %}
+***
+
+### Recommended Skills
+
+For this tutorial, we recommend that you have some basic familiarity with the HDF5 file format, including knowing how to open HDF5 files (in Rstudio or HDF5Viewer) and how groups and metadata are structured. To brush up on these skills, we suggest that you work through the <a href="{{ site.baseurl }}/intro-hdf5-r-series" target="_blank"> *Introduction to Working with HDF5 Format in R* series</a> before moving on to this tutorial.
+
+</div>
+
+
+## Why subset your dataset? 
+There are many reasons why you may wish to subset your HDF5 file. 
+Primarily, HDF5 files may contain a large amount of information 
+that is not necessary for your purposes. By subsetting the file, 
+you can reduce file size, thereby shrinking your storage needs, 
+shortening file transfer/download times, and reducing your processing 
+time for analysis. In this example, we 
+will take a full HDF5 file of NEON hyperspectral reflectance data 
+from the San Joaquin Experimental Range (SJER) that has a file size 
+of ~652 Mb and make a new HDF5 file with a reduced spatial extent, 
+and a reduced spectral resolution, yeilding a file of only ~50.1 Mb. 
+This reduction in file size will make it easier and faster to conduct 
+your analysis and to share your data with others. We will then use this 
+subsetted file in the <a href="{{ site.baseurl }}/intro-hsi-r-series" target="_blank"> *Introduction to Hyperspectral Remote Sensing Data* series</a>.
+
+If you find that downloading the full 652 Mb file takes too much 
+time or storage space, you will find a link to this subsetted file 
+at the top of each lesson in the <a href="{{ site.baseurl }}/intro-hsi-r-series" target="_blank"> *Introduction to Hyperspectral Remote Sensing Data* series</a>.
+
+## Exploring the NEON hyperspectral HDF5 file structure
+
+In order to determine what information that we want to put into our subset, we should first take a look at the full NEON hyperspectral HDF5 file structure to see what is included. To do so, we will load the required package for this tutorial (you can un-comment the middle two lines to load 'BiocManager' and 'rhdf5' if you don't already have it on your computer).
```

Comment:
Throughout you have many lines >80 characters.  Can you break them at 80 chars so on IDEs that don't automatically wrap text they won't run off the screen?

Reply:
Ahhhhh okay now I understand the reason for doing this - IDEs that don't do the wrapping automatically! I'm sure that you had told me this before but I forgot. Thanks for the reminder. I have updated this throughout the tutorial.